packages/js-config-webpack-plugin/src/JsConfigWebpackPlugin.js

```diff
@@ -33,13 +33,40 @@ class JsConfigWebpackPlugin {
 	 * @returns {string}
 	 */
 	resolveBabelConfigFilePath(contextPath, environmentName) {
+		/**
+		 * Return the value, unless its an interator, in which case get the last value.
+		 * Works around differences in return values of @babel/core functions between versions.
+		 * @template T
+		 * @param {Generator<T, T> | T} valueOrIterator
+		 * @returns {T}
+		 */
+		function getResult(valueOrIterator) {
+			if (!isGenerator(valueOrIterator)) {
+				return valueOrIterator;
+			}
+			let result = valueOrIterator.next();
+			while (!result.done) {
+				result = valueOrIterator.next();
+			}
+			return result.value;
+		}
+
+		/**
+		 * Returns true if the value looks like a generator.
+		 * @param {any} val
```

Comment:
use `unknown` as type here instead? :)

Reply:
`unknown` caused issues there - so I fall back to unknown